aws/src/main/java/org/apache/iceberg/aws/glue/IcebergToGlueConverter.java

```diff
@@ -234,8 +258,26 @@ static void setTableInputInformation(
       Optional.ofNullable(properties.get(GLUE_DESCRIPTION_KEY))
           .ifPresent(tableInputBuilder::description);
```

Comment:
Hi @lawofcycles thank you for your patch!

Now this method has the existing table information, so I think it would be better to move the function to keep the table description https://github.com/aajisaka/iceberg/blob/58c61241bd61de7a4062dbf664691f05eeb2ea53/aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java#L319-L321 into this method.

Reply:
Thank you for your review!
I moved keeping the table description logic to IcebergToGlueConverter.